JavaAssignment/src/ravi/Addition.java

```diff
@@ -0,0 +1,20 @@
+package ravi;
+
+//Class with same method name to add integers,decimal and strings
+public class Addition {
+	
+	void add(int number1,int number2){			//method to add 2 integers
+		int result=number1+number2;
+		System.out.println("Sum is "+result);
+	}
+	
+	void add(float number1,float number2){		//method to add 2 decimals
+		float result=number1+number2;
+		System.out.println("Sum is "+result);
+	}
+	
+	void add(String string1,String string2){		//method to add 2 strings
```

Comment:
here u concat the string1 & string2 not making sum of it
you need to parse that value in Integer and then convert it into string value.